### PR TITLE
Getting started page is added in docs/guides

### PR DESCRIPTION
**Pull request by pushpendupurkait**
```
In this page we are showing some steps to check node version and running
JavaScript code with Node.js
```

### CODE REVIEW

**Comment by pushpendupurkait** [comment]
@fhemberger , please take a look at the page and inform me if further changes are required.

File: locale/en/docs/guides/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Getting Started
+layout: docs.hbs
+---
+# Getting Started
+
+First we have to make sure that Node.js is correctly installed in you
```
Comment (ghost) [inline locale/en/docs/guides/getting-started.md]: *on your system

File: locale/en/docs/guides/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Getting Started
+layout: docs.hbs
+---
+# Getting Started
+
+First we have to make sure that Node.js is correctly installed in you 
+system.
+Then we will run a small JavaScript code using node.
```
Comment (ghost) [inline locale/en/docs/guides/getting-started.md]: `Then we will run a short piece of JavaScript code using node`

File: locale/en/docs/guides/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+**Test node:** To check if Node is installed properly in you system,
```
Comment (ghost) [inline locale/en/docs/guides/getting-started.md]: again, `on your system`

File: locale/en/docs/guides/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+open the terminal or command prompt and run command 
+<code>node -v </code>. This should print the version, so it will 
+show you something like this <code>v4.5.0</code>.
+
+```javascript
+c:\node> node -v
+v4.5.0
+```
+
+**Create a test file and run it:** This is the easiest way to check Node.js
```
Comment (ghost) [inline locale/en/docs/guides/getting-started.md]: `to check if Node.js is installed on your system`

**Review by ghost — CHANGES_REQUESTED** [review]
please fix these spelling mistakes, thank you!

**Comment by pushpendupurkait** [comment]
@fene , Sure
I'll make changes according to the comments. 
Thanks for the feedback

**Comment by pushpendupurkait** [comment]
Hi, @fene 
I have made changes as suggested by you. please review it.

**Comment by pushpendupurkait** [comment]
@fene 
Is there any other changes for the document ?

**Comment by Trott** [comment]
I don't think this should be posted as a "Getting Started" page and I don't think it is sufficiently in depth to qualify as a guide. This content seems more like two steps one should take after installing. We don't seem to have an installation page (do we?) so maybe one should be created and this should be integrated into it.

At a minimum, if it's going to be a "Getting Started" page, it:
- should include a handful of simple but useful examples, such as one sees at https://howtonode.org/hello-node (Not all the examples on that page would be appropriate, although I think the first few might be along the lines of something we'd want.)
- should include a short list of places to go next to learn more about Node.js. Maybe a link to nodeschool.io, the API docs, something about working with npm, and a few similar resources or something like that.

So I think there are two options: Include this content as part of a larger "installation" page or expand it to make it more substantial.

**Comment by fhemberger** [comment]
Agreeing with @Trott, please feel free to submit a more comprehensive version.

**Pull request closed without merging**